openupgradelib/openupgrade.py

```diff
@@ -1721,9 +1721,17 @@ def logged_query(cr, query, args=None, skip_no_result=False):
         duration = datetime.now() - start
         if log_msg:
             try:
-                full_query = tools.ustr(cr._obj.query)
+                if version_info[0] < 18:
+                    full_query = tools.ustr(cr._obj.query)
+                else:
+                    full_query = cr._obj.query.decode("utf-8", errors="strict")
```

Comment:
Thanks! We keep a small number of functions with a version depending definition at the top of this file (around line 123). How about adding a custom `decode` function to them?

Reply:
Good idea, but how to "import" python `decode` method? 
Doing something like 
```
if version_info[0] < 18:
    custom_decode = tools.ustr
else:
    custom_decode = decode
```
won't work because `decode` is not defined?

Reply:
I would just call it `decode`, so something like

```
decode = lambda s: s.decode("utf-8", errors="strict")
```

Reply:
Done